genetics-research.html

```diff
@@ -13,12 +13,21 @@ <h1 class="hero__heading">
                 </h1>
                 <div class="hero__content wysiwyg">
                     <p>
-                        The tskit ecosystem can be used for a wide range of genetic research purposes. The tree sequence
-                        file format is not only an efficient, scalable way to store and analyse genotype data, but as
-                        its underlying structure is a set of one or more trees it can be used to store and analyse phylogenetic data.
+                        Tskit enables rapid and principled genomic analysis. The tree
+                        sequence file format provides an optimised, scalable way to
+                        store genetic variation data, and underlies modern whole-genome
+                        simulation software such as
+                        <a href="software/msprime.html">msprime</a>. This is achieved
+                        by focussing on the evolutionary trees that underlie DNA
+                        sequence data, and providing efficent methods to analyse them.
+                        Tskit therefore provides a framework that serves to unify the
```

Comment:
That's a big claim that might annoy people. How about "Tskit provides a common software framework that can be used across the fields of X Y and Z"?

Reply:
Good point. That's a nice phrasing